### PR TITLE
Feature: USWDS update and remove node sass

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Related to: 

- https://github.com/GSA/datagov-deploy/issues/2566
- https://github.com/GSA/datagov-deploy/issues/2807
- https://github.com/GSA/datagov-deploy/issues/2808

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -50,62 +38,6 @@ ignore:
...
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540978:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540980:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
 # patches apply the minimum changes required to fix a vulnerability
```
Comment (adborden) [inline .snyk:41]: You should remove these patches. I'm not seeing `snyk patch` in the package.json scripts, so these could be hiding true positives.

File: gulpfile.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 var autoprefixer = require("autoprefixer");
-var autoprefixerOptions = require("./node_modules/uswds-gulp/config/browsers");
-var cssnano = require("cssnano");
+// var autoprefixerOptions = require("./node_modules/uswds-gulp/config/browsers");
```
Comment (adborden) [inline gulpfile.js:2]: You can delete this.
```suggestion
```

**Review by adborden — APPROVED** [review]
👏  excellent!

**Comment by jbrown-xentity** [comment]
I removed gulp-svg-sprite, even though it was [required by uswds-gulp](https://github.com/uswds/uswds-gulp#requirements). Site built and seems to work fine, although someone should check if they know what page a sprite might exist on...
Removed because it had dependencies that were old/deprecated, and no great remediation path available.

**Pull request merged**